openreview/conference/templates/reviewerWebfield.js

```diff
@@ -190,7 +189,7 @@ var getCustomLoad = function(userIds) {
     return $.Deferred().resolve(0);
   }
 
-  return Webfield.get('/notes', { invitation: CUSTOM_LOAD_INVITATION })
+  return Webfield.get('/notes', { invitation: CUSTOM_LOAD_INVITATION, select: 'content.reviewer_load,content.user' })
```

Comment:
Are we still using notes for custom reviewer loads? Or is this only going to be stored in edges?

Reply:
we are still using a note to save the reduced load and the edges are created later. However, the PCs are starting to create the edge by themselves, so I guess we should check both.